training-node/app/controllers/user.js

```diff
@@ -5,7 +5,8 @@ const userServices = require('../services/user'),
   bcrypt = require('bcrypt'),
   tokenManager = require('../services/tokenManager'),
   validations = require('./validations'),
-  SALT_ROUNDS = 10;
+  SALT_ROUNDS = 10,
```

Comment:
we usually have a constants file, for the future.

Reply:
Ok :+1: